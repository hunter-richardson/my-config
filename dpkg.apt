albert
apcalc
apt
apt-fast
apt-transport-https
aptitude
build-essential
caffeine
chrome-gnome-shell
cmus
cubic
curl
dconf-editor
default-jre
docky
dos2unix
dtrx
duplicity
exuberant-ctags
file
fish
flat-remix
flat-remix-gnome
fonts-hack-ttf
gdebi
git
gnome-paint
gnome-shell-extension-weather
gnome-shell-mailnag
gnome-tweak-tool
google-chrome-stable
googler
gparted
grc
gtk2-engines-murrine
gtk2-engines-pixbuf
libreoffice-script-provider-python
libwayland-egl1
lxd
lynx
members
mesa-utils
mlocate
moreutils
htop
mps-youtube
ocfs2-tools
pastebinit
pithos
python3-pip
python3-dev
ruby-full
secure-delete
snapd
source-highlight
steghide
sublime-text
symlinks
tlp
tlp-rdw
tmux
ubuntu-restricted-extras
urlview
w3m
wine-stable
wordgrinder
xsel
