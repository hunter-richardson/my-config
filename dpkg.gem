bropages
gem-man
manpages
rubygems-update
