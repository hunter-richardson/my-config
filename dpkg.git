hunter-richardson/my-config
decors/fish-colored-man
decors/fish-highlight
decors/fish-source-highlight
evanyeung/terminal-slack
PromyLOPh/pianobar
tmux-plugins/tpm
tmux-plugins/tmux-logging
tmux-plugins/tmux-prefix-highlight
tmux-plugins/tmux-urlview
tmux-plugins/tmux-yank
