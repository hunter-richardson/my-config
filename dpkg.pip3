click
click-didyoumean
click-man
click-spinner
hangups
thefuck
