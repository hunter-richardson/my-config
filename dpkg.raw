https://raw.githubusercontent.com/jarun/googler/master/googler
