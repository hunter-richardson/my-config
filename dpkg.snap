communitheme
cheat
discord
google-play-music-desktop-player
gnome-system-monitor
tldr

